tests/tests/test_train_schedule.py

```diff
@@ -1253,7 +1253,7 @@ def test_etcs_schedule_braking_curves_endpoint(
     ]
     start_braking_curves_offsets = [
         [21_467_192, MAX_SPEED_288],
-        [40_663_497, SPEED_LIMIT_142],
+        [34_638_530, MAX_SPEED_288],
```

Comment:
I don't get this change quickly.

Reply:
With the new stop locations, the svl impacts the braking curve and it skips the train's max envelope curve at 142km/h and intersects with it earlier on the path and higher in the speed limit, at 288km/h.

Reply:
Oh, this is actually an interesting case now: the brake curve is hitting the LoA maintain-speed, but the information curve doesn't.
This is what we could expect from specifications and it looks OK.
But if we ever want to evolve specifications and "integrate LoA maintain-speed to MRSP when computing info-curves", it would be alright to have it change too (and we would have the test for this).

So I would note it:
```suggestion
        [34_638_530, MAX_SPEED_288], # hitting the LoA maintain-speed, but not the actual MRSP
```